core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/util/base/RightV4.java

```diff
@@ -19,4 +31,9 @@
     public AuthorizationResourceAction getAction() {
         return action;
     }
+
+    public static RightV4 getByAction(AuthorizationResourceAction authorizationResourceAction) {
```

Comment:
Can you please use a static final enum map for the lookup?

Reply:
same here, noone can guarantee that action field is not the same for two enum value

Reply:
fixed